matrix_sdk_base/src/store/memory_store.rs

```diff
@@ -311,6 +318,39 @@ impl MemoryStore {
             .get(room_id)
             .and_then(|m| m.get(event_type.as_ref()).map(|e| e.clone())))
     }
+
+    async fn set_media_content(&self, request: &MediaRequest, data: Vec<u8>) -> Result<()> {
```

Comment:
i think `add_media_content` or `store_media_content` would be a better fitting name.